MID Server/MID Server Script Includes/IllumioManageAsyncJobs.js

```diff
@@ -36,12 +36,19 @@ IllumioManageAsyncJobs.prototype = {
 
         this.protocol = this.utils.getPortFromUrl(this.pceUrl);
         
+        this.retryParams = null;
```

Comment:
Same issue here as in IllumioAuthenticateToPCE. Why are we duplicating this logic? Can we put it in a shared function or move it to the `IllumioHTTPClient` init?

Reply:
retryParams i can remove, else other things are different for different classes so